libs/Daux.php

```diff
@@ -90,6 +93,18 @@ public function initializeConfiguration($override_file = 'config.json')
         }
     }
 
+    public function setThemesPath($path)
+    {
+        $this->themes_path = $path;
+        if (!is_dir($this->themes_path) &&
+            !is_dir($this->themes_path = $this->local_base . DIRECTORY_SEPARATOR . $this->themes_path)
+        ) {
+            throw new Exception('The Themes directory does not exist. Check the path again : ' . $this->themes_path);
+        }
+        $this->options['themes_path'] = $this->themes_path;
+        $this->options['templates'] = $this->themes_path . DIRECTORY_SEPARATOR . 'core' . DIRECTORY_SEPARATOR . 'templates';
```

Comment:
Yes and no, I think we can go even further with the concept.

The `resources` and `templates` folder were separated for a reason : the templates are the default templates for the system, if we change the structure / add features, it will automatically come included with the .phar archive when you update.

The resources however, are not part of the "default" rendering, they come apart.

So when you want to create a new theme, it can be a theme only for the style, meaning you don't need to add the templates to your folder.

Or it can be a theme for the whole system, and only in this case you would include the template overrides.